---
int64 board_length
int64 board_width
int64 ball_speed
